sktime/forecasting/arch.py

```diff
@@ -48,14 +48,13 @@ def __init__(
 
         super().__init__()
 
-    def _instantiate_model(self):
-        # import inside method to avoid hard dependency
+    def _get_statsforecast_class(self):
         from statsforecast.models import GARCH as _GARCH
 
-        return _GARCH(
-            p=self.p,
-            q=self.q,
-        )
+        return _GARCH
+
+    def _get_statsforecast_params(self):
```

Comment:
we could also make this abstract, and default to `return self.get_params()` - but leave the implementer an option to override.

Reply:
done